solidity/contracts/xMEZO.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0
+//
+//  __       __  ________  ________   ______  
+// |  \     /  \|        \|        \ /      \ 
+// | $$\   /  $$| $$$$$$$$ \$$$$$$$$|  $$$$$$\
+// | $$$\ /  $$$| $$__        /  $$ | $$  | $$
+// | $$$$\  $$$$| $$  \      /  $$  | $$  | $$
+// | $$\$$ $$ $$| $$$$$     /  $$   | $$  | $$
+// | $$ \$$$| $$| $$_____  /  $$___ | $$__/ $$
+// | $$  \$ | $$| $$     \|  $$    \ \$$    $$
+//  \$$      \$$ \$$$$$$$$ \$$$$$$$$  \$$$$$$ 
+                                           
+pragma solidity 0.8.29;
+
+import "./mERC20.sol";
+
+// @title xMEZO
+// @notice Cross-chain representation of the MEZO token.
+// @dev MEZO token is a native precompile of the Mezo chain. 
+//      This contract serves as a representation of the MEZO 
+//      token on foreign EVM chains for bridging purposes.
+contract xMEZO is mERC20 {}
```

Comment:
Can we call it just MEZO? This will be less confusing for users of other chains.

Reply:
Reasoning was to not confuse with https://github.com/mezo-org/mezod/blob/main/precompile/mezotoken/IMEZO.sol which is implemented as precompile and clearly indicate this is cross-chain MEZO. I can change it if you guys feel it's confusing. From my perspective having two MEZO implementations in the same repo may be confusing even more but I may be biased.

PS: [I tried to align on the naming before writing the first line](https://discord.com/channels/1079490158639976609/1445088307540136046/1448284553495187650) of code exactly to avoid comments like this 😄